README.md

```diff
@@ -138,92 +133,45 @@ ReactDOM.render(<App person={person} />, document.body)
 person.name = "Mike" // will cause the Observer region to re-render
 ```
 
-### `useLocalStore` hook
+### `useLocalObservable` hook
 
 [User guide](https://mobx-react.js.org/state-local)
 
-Local observable state can be introduced by using the `useLocalStore` hook, that runs once to create an observable store. A quick example would be:
+Local observable state can be introduced by using the `useLocalObservable` hook, that runs once to create an observable store. A quick example would be:
 
 ```javascript
-import { useLocalStore, useObserver } from "mobx-react-lite"
+import { useLocalObservable, Observer } from "mobx-react-lite"
 
 const Todo = () => {
-    const todo = useLocalStore(() => ({
+    const todo = useLocalObservable(() => ({
         title: "Test",
         done: true,
         toggle() {
             this.done = !this.done
         }
     }))
 
-    return useObserver(() => (
-        <h1 onClick={todo.toggle}>
-            {todo.title} {todo.done ? "[DONE]" : "[TODO]"}
-        </h1>
-    ))
+    return (
+        <Observer>
+            {() => (
+                <h1 onClick={todo.toggle}>
+                    {todo.title} {todo.done ? "[DONE]" : "[TODO]"}
+                </h1>
+            )}
+        </Observer>
+    )
 }
 ```
 
-When using `useLocalStore`, all properties of the returned object will be made observable automatically, getters will be turned into computed properties, and methods will be bound to the store and apply mobx transactions automatically. If new class instances are returned from the initializer, they will be kept as is.
+When using `useLocalObservable`, all properties of the returned object will be made observable automatically, getters will be turned into computed properties, and methods will be bound to the store and apply mobx transactions automatically. If new class instances are returned from the initializer, they will be kept as is.
```

Comment:
Methods will be true actions in MobX6, wouldn't? 

But personally, I would trim these docs down a lot only to the API summary and let http://mobx.js.org/ as central point for any explanations. It's unnecessarily duplicated this way imo.

Reply:
Correct